doc/sphinx/install.rst

```diff
@@ -55,12 +55,12 @@ Installation instructions for the web frontend
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 First, install the following dependencies (debian packages)
-``postgresql postgresql-server-dev-all postgresql-client python-virtualenv build-essential libxml2-dev libxslt1-dev python-dev gettext libjpeg-dev libffi-dev``
+``postgresql postgresql-server-dev-all postgresql-client python3-venv build-essential libxml2-dev libxslt1-dev python3-dev gettext libjpeg-dev libffi-dev``
 
 Then, build a virtual environment to isolate all the python
 dependencies::
 
-   virtualenv .virtualenv
+   python3 -m venv .virtualenv
```

Comment:
I updated the provisioning script recently and am using the `python3-virtualenv` package (https://github.com/dissemin/dissemin/blob/master/provisioning/provision.sh#L27). Do you know what is the difference with `python3-venv`? Thanks!

Reply:
I think `python3 -m venv` is recommended over the `virtualenv` command as it runs in python directly. I think the latter is even sort of deprecated.